config/kernel/linux-rk35xx-vendor.config

```diff
@@ -8040,19 +8119,24 @@ CONFIG_TSL2583=y
 #
 # Magnetometer sensors
 #
-# CONFIG_AK8974 is not set
-# CONFIG_AK8975 is not set
-# CONFIG_AK09911 is not set
-# CONFIG_BMC150_MAGN_I2C is not set
-# CONFIG_BMC150_MAGN_SPI is not set
-# CONFIG_MAG3110 is not set
-# CONFIG_MMC35240 is not set
-# CONFIG_IIO_ST_MAGN_3AXIS is not set
-# CONFIG_SENSORS_HMC5843_I2C is not set
-# CONFIG_SENSORS_HMC5843_SPI is not set
-# CONFIG_SENSORS_RM3100_I2C is not set
-# CONFIG_SENSORS_RM3100_SPI is not set
-# CONFIG_YAMAHA_YAS530 is not set
+CONFIG_AK8974=m
+CONFIG_AK8975=m
```

Comment:
Should be disabled, because make a conflict with another sensors:
[🔨]   error: the following would cause module name conflict:
[🔨]     drivers/input/sensors/compass/ak8975.ko
[🔨]     drivers/iio/magnetometer/ak8975.ko

Reply:
i find another way to fix this, i renamed the modules